### ISSUE TITLE
Hi may I ask if this is tested for docker version of hass?

### ISSUE THREAD

**Issue by pretubedongit** [shing6326/hacs-zb-ir01-to-climate#1]
Hi may I ask if this is tested for docker version of hass? Is it possible to use in with dockerized HASS? I have mounted CC2652 and it's connected to the IR01.

**Comment by shing6326** [comment]
Hello, the plugin only reads the `last_received_command` of the IR01 entity which you specified from the config, and use the set the `send_command` entity to control the AC. I have never use docker version of HASS but it would work as long as the IR01 entities are recognized by the system

**Comment by pretubedongit** [comment]
Thank you for your reply. My server is Debian 12 headless. I tested the docker version. Problem is that docker HA is only built in with ZHA, not Z2M, and no addons are allowed (because addons are basically dockers also). In theory it should be possible to just run docker ha, docker mqtt and docker z2m and make the 3 docker containers talk to each other through LAN, but I only got docker ha and docker mqtt up, but I could not find any way to run docker z2m on debian 12.

This easyiot IR remote seems not supporting ZHA command sending. But it works well with Z2M. Please correct my if I am wrong.

 So in the end I switched to KVM HA supervised, hosted on my Debian 12, and got ZHA installed. Haven't been able to configure your frontend climate add on but it looks promising.

Could you share more on how you run the HA together with this addon? Does your IR blaster zigbee connect with HA with ZHA or Z2M or something else? Do you run them dockerized or bare metal, or kvm, etc?

**Comment by shing6326** [comment]
I am using the HAOS install. The IR01 should work on Z2M with latest release, you could use any kind of HA installation methods which Z2M supports.

**Comment by pretubedongit** [comment]
Hi I am able to use the MQTT or Z2M frontend to send command. However I could not find your integration in my 'Integration' page. 

What I did:
1. I made a folder under config, custom_components, and uploaded the folder.
2. I copied the config to configuration.yaml.
```
zb-ir01-to-climate:
  - ir01_entity_id: 0xf4b3b1fffe132df2
    climate_id: climate.my_ac
    climate_name: My AC
```
4. Rebooted HAOS. Did not show in Integrations.
5. I tried to change the ir01_entity_id to "sensor.ac_remote_last_received_command" or "text.ac_remote_send_command", both not helping.

<img width="1377" alt="image" src="https://github.com/shing6326/hacs-zb-ir01-to-climate/assets/155866580/86a70674-ffa8-4856-8a9c-8c73b9b705cb">


I could search it but cannot add. It showed 
>"You can add this device by adding it to your 'configuration.yaml'. See the documentation for more information."

Here is where I put the files

<img width="895" alt="image" src="https://github.com/shing6326/hacs-zb-ir01-to-climate/assets/155866580/d497f5f1-3750-4e58-a977-87186560858c">


Did I miss anything? Should I make any other changes?
